bindings/gumjs/gumv8database.cpp

```diff
@@ -163,8 +165,15 @@ GUMJS_DEFINE_FUNCTION (gumjs_database_open)
   if (!_gum_v8_args_parse (args, "si", &path, &flags))
     return;
 
+  auto interceptor = core->script->interceptor.interceptor;
   handle = NULL;
+
+  gum_interceptor_ignore_current_thread (interceptor);
```

Comment:
Ah, just remembered that we have `GumV8InterceptorIgnoreScope` in `gumv8platform.cpp` -- would be great to move that to `gumv8scope.h` and make it take `GumV8Core *` as a constructor argument, and then use it here and elsewhere to simplify the code.